### ISSUE TITLE
fullscreen button in examples is broken in IE9

### ISSUE THREAD

**Issue by RandomEtc** [simplegeo/polymaps#13]
Broken = the placement is wrong. 

In IE9 it doesn't look like the styling is taking hold (position, left and top are all blank).

In Firefox it is almost positioned correctly but the alignment is off and it disappears on full screen. I suspect the DOCTYPE or the addition of overflow:hidden to all svg elements in the CSS probably threw it off.

**Comment by RandomEtc** [comment]
It seems like it's also misplaced in Firefox in the ie9 branch - possibly due to the DOCTYPE?

**Comment by mbostock** [comment]
Yeah, I've fixed this now. The problem is that I was setting the style property to -16, when I should have been setting it to "-16px".
